README.md

```diff
@@ -16,34 +16,42 @@ git checkout 1_offering-passkeys
 
 ## Setup
 
-These instructions will assume you're running the server locally. See the [Docker](#docker) section for information on how to run it anywhere.
+The following instructions will assume you're running the server locally. See the [Docker](#docker) section for production environments.
 
-### Environment
+**Prerequisites for running locally**
+1. Install [Node](#node)
+1. Install [dependencies](#dependencies)
+1. Create a local `.env` file or rename [.sample.env](#.sample.env) to `.env`
 
-Export the environment variables specified in [CONFIG](./CONFIG.md). For local development, you can use a `.env` file.
```

Comment:
Let's keep this description, but I like the idea of incorporating the use of the `.sample.env` file. So maybe modify the line to something like:

```
Export the environment variables specified in [CONFIG](./CONFIG.md). For local development, you can use a `.env` file, which can easily be created by copying the [.sample.env](./.sample.env) file to `.env`.
```